app/scripts/components/common/catalog/catalog-tags.tsx

```diff
@@ -39,9 +39,10 @@ const CatalogTagsContainer: React.FC<CatalogTagsContainerProps> = ({
 }) => {
   if (allSelectedFilters.length > 0 || urlTaxonomyItems.length > 0) {
     return (
-
-
-      <div className='display-flex flex-wrap tablet:margin-bottom-3'>
+      <div
+        className='display-flex flex-wrap tablet:margin-bottom-2'
+        style={{ rowGap: '8px', columnGap: '8px' }}
```

Comment:
Have you considered to use the Grid components? https://trussworks.github.io/react-uswds/?path=/docs/components-grid--docs